engine_test.go

```diff
@@ -706,48 +706,93 @@ func TestStarPanic197(t *testing.T) {
 }
 
 func TestIndexes(t *testing.T) {
-	require := require.New(t)
```

Comment:
why did you remove this?

Reply:
Because now it's created inside each of the subtests. Otherwise, `require` is redefined and you can't do `require.New` inside